rs/src/gtest/calls.rs

```diff
@@ -125,6 +125,7 @@ impl Remoting for GTestRemoting {
         code_id: CodeId,
         salt: impl AsRef<[u8]>,
         payload: impl AsRef<[u8]>,
+        _gas_limit: Option<GasUnit>,
```

Comment:
Either raise a separate issue for gtest or remove `resolves #...` from this one, maybe replacing it with `towards #...`